twine-upload.sh

```diff
@@ -40,6 +40,13 @@ INPUT_VERIFY_METADATA="$(get-normalized-input 'verify-metadata')"
 INPUT_SKIP_EXISTING="$(get-normalized-input 'skip-existing')"
 INPUT_PRINT_HASH="$(get-normalized-input 'print-hash')"
 
+PASSWORD_DEPRECATION_NUDGE="::error title=Password-based uploads deprecated::\
+Starting in 2024, PyPI will require all users to enable Two-Factor \
```

Comment:
@woodruffw FYI looks like it's time to drop that future tense from the error message. So I filed #233 for when somebody gets to it.

Reply:
Progress! 😄 

I'll go ahead and do it now, since it's a tiny change 🙂